workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -236,6 +211,17 @@ def _print(self, message: str):
             print(message)
 
 
+def state_updates_from_tendency(state_updates_from_tendency,):
```

Comment:
Name of function and arg the same.

I think this is fine, we have talked about trying to make the data flow clearer, but I don't think we came up with a solution without a more holistic refactoring of the loop.